README.md

```diff
@@ -56,7 +56,7 @@ unzip bin.zip
 ```
 
 - CD to docker folder `cd docker`
-- Run `make c2wasm-api && make clangd && make wasi-sdk && make hook-cleaner`
+- Run `make c2wasm-api && make clangd && make wasi-sdk && make hook-cleaner && make c2wasm-api/clang/includes/src/ripple/app/hook`
```

Comment:
This executes the next Makefile, but this compilation is dependent on the local environment. This is useless if the local and container architectures are different.

`c2wasm-api/clang/includes/src/ripple/app/hook/`

```
guard_checker: guard_checker.cpp Guard.h Enum.h
	g++ -o guard_checker guard_checker.cpp --std=c++17 -g
install: guard_checker
	cp guard_checker /usr/bin/
```